src/nvim/main.c

```diff
@@ -374,6 +374,8 @@ int main(int argc, char **argv)
     filetype_maybe_enable();
     // Sources syntax/syntax.vim, which calls `:filetype on`.
     syn_maybe_enable();
+    // Sources autocmds.vim
+    init_default_autocmds();
```

Comment:
This should go after init_default_mappings() (or we should treat them the same way whatever we decide). I do not consider these default autocmds to be a "plugin", just like default mappings and default options are part of `nvim -u NONE`.